backend/test_data.py

```diff
@@ -3,110 +3,110 @@
 import string
 from datetime import datetime
 
+
+def users(num, start=0):
+    '''
+    Generates a list of usernames for upvote/downvote lists
+    '''
+    return ['user' + str(i) for i in range(start, start+num)]
+
+
 # valid
 price0 = model.Price(
     user='admin',
-    upvote=10,
-    downvote=5,
+    upvotes=users(10),
+    downvotes=users(5, 10),
     price=1.99,
     date=datetime.now().timestamp()
 )
 
 # valid
 price1 = model.Price(
     user='admin',
-    upvote=7,
-    downvote=4,
+    upvotes=users(7),
+    downvotes=users(4, 7),
     price=0.99,
     date=datetime.now().timestamp()
 )
 
 # valid
 price2 = model.Price(
     user='admin',
-    upvote=0,
-    downvote=0,
+    upvotes=users(0),
+    downvotes=users(0, 0),
```

Comment:
Line 35 and 36 both return an empty array correct? "[]" might be simpler to understand